be/src/block_cache/cache_options.cpp

```diff
@@ -111,4 +113,16 @@ Status parse_conf_datacache_disk_spaces(const std::string& config_disk_path, con
     return Status::OK();
 }
 
+void clear_residual_datacache(const std::string& disk_path) {
+    auto st = FileSystem::Default()->iterate_dir2(disk_path, [&](DirEntry entry) {
```

Comment:
This `st` is actually not used？

Reply:
> This `st` is actually not used？

Changed, even if the clean error is reported here, it does not have a significant impact and there is no need to block BE startup, so we just print an error log here.